src/bitcoin-cli.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include <chainparamsbase.h>
 #include <clientversion.h>
+#include <netaddress.h>
```

Comment:
It seems `netaddress.cpp` is part of `LIBBITCOIN_COMMON`, which is not linked into `bitcoin-cli`. So you're not currently allowed to use this here.

Reply:
Thanks! I forgot to look at the makefile. Moved them to `httpserver.h`, which seems to be allowed, IIUC.

Reply:
Oops, "A new circular dependency in the form of "httpserver -> netbase -> netaddress -> httpserver" appears to have been introduced." Will create a header, I guess.